dev/cassandra-server/src/main/scala/com/lightbend/lagom/internal/cassandra/CassandraLauncher.scala

```diff
@@ -23,24 +23,42 @@ class CassandraLauncher {
   private val ForcedShutdownTimeout = 20.seconds
   private var cassandraDaemon: Option[Closeable] = None
 
+  private val devEmbeddedYaml = "dev-embedded-cassandra.yaml"
+
   var port: Int = 0
+
   def address = s"$hostname:$port"
+
   def hostname = "127.0.0.1"
 
-  def start(cassandraDirectory: File, configResource: String, clean: Boolean, port: Int, jvmOptions: Array[String]): Unit = this.synchronized {
+  def start(
+    cassandraDirectory: File,
+    yamlConfig:         File,
+    clean:              Boolean,
+    port:               Int,
+    jvmOptions:         Array[String]
+  ): Unit = this.synchronized {
+
     if (cassandraDaemon.isEmpty) {
+
       prepareCassandraDirectory(cassandraDirectory, clean)
 
       val storagePort = AkkaCassandraLauncher.freePort()
-
+      val fileOrResource =
+        Option(yamlConfig)
```

Comment:
This is related to the structural typing I mentioned before. We remove the option to reintroduce it here. Smelly